lms/djangoapps/verify_student/management/commands/approve_id_verifications.py

```diff
@@ -149,5 +151,37 @@ def _approve_id_verifications(self, user_ids):
         for verification in existing_id_verifications:
             verification.approve(service='idv_verifications command')
             send_approval_email(verification)
+            self._update_verified_name(verification)
 
         return list(failed_user_ids)
+
+    def _update_verified_name(self, verification):
```

Comment:
I don't like how this function is named. Should we name this more specifically? For example: `approve_verified_name_by_software_secure_verifications`?